HealthSDK/GiniHealthSDK/Documentation/source/Integration.md

```diff
@@ -45,29 +45,9 @@ private class MyAlternativeTokenSource: AlternativeTokenSource {
 
 ## Certificate pinning (optional)
 
-If you want to use _Certificate pinning_, provide metadata for the upload process, you can pass both your public key pinning configuration as follows:
+If you want to use _Certificate pinning_, provide metadata for the upload process, you can pass both your public key pinning configuration for more information)
```

Comment:
[clarification]: There is an `)` after information. What should be here? And to me, it seems the sentence is unfinished.